src/transformers/models/luke/tokenization_luke.py

```diff
@@ -1378,6 +1483,171 @@ def _pad(
 
         return encoded_inputs
 
+    # Copied from transformers.models.roberta.build_inputs_with_special_tokens
```

Comment:
```suggestion
    # Copied from transformers.models.roberta.build_inputs_with_special_tokens with RoBERTa->LUKE
```
So the RoBERTa in the docstring is adapted to LUKE.